mesheryctl/pkg/utils/healthcheck.go

```diff
@@ -114,6 +115,12 @@ func parseKubectlShortVersion(version string) ([3]int, error) {
 
 // IsMesheryRunning checks if the meshery server containers are up and running
 func IsMesheryRunning(currPlatform string) (bool, error) {
+	resp, err := http.Get("http://localhost:9081/api/server/version")
```

Comment:
```suggestion
	resp, _ := http.Get("http://localhost:9081/api/server/version")
```
Mark it like this.

Reply:
Also, add a comment above this
// Checking if Meshery is running with "make run-local" or "make run-fast"

Reply:
add a comment over this,
// checking if meshery is running with "make run-local" or "make run-fast"

Reply:
Hmm alright! That will do 👍